src/main/java/duke/gui/MainWindow.java

```diff
@@ -0,0 +1,65 @@
+package duke.gui;
+
+import duke.command.Command;
+import duke.main.Duke;
+import duke.main.DukeException;
+import duke.main.Parser;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
```

Comment:
Perhaps you could separate this group of import statements into smaller groups for readability? I notice the same thing in other places as well.